_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title:  "Introduction to system tests"
+date: 2020-09-07
+author: maroš_orsák
+---
+
+Lessons learn from past years shown that testing is the essential part of the software cycle. We have known many 
+failures that happened because of neglect testing. In Strimzi, we exactly know how we should test the features. 
+Follow me on this blog post and see the basic view of our tests.
+
+In this blog post we are gonna look at system tests and I will give you motivation why you should look at it. The content 
+is as follows:
+
+# Content
+1. [Motivation](#Resources)
+2. [How they differ from our UT and IT](#Lifecycle of tests)
+3. [How they run in relation with Kubernetes](#Auxiliary classes)
+4. [How to run them](#How to create a system test)
+5. [Conclusion](#Conclusion)
```

Comment:
Not sure the content is needed. We do not use it for other posts. I don't mind if you want to have it here. But I think it can be also left out.

Reply:
Hmm, I am a supporter of the content because it shows the reader whole writing.  I would prefer to let content but if Paolo and you think that's should not be used I will remove it :(.